client/packages/common/src/ui/icons/Eye.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const EyeIcon = (props: SvgIconProps): JSX.Element => (
```

Comment:
the icons are taken from feather, so they differ from those used in HSH.
We are using feather icons, if there are icons available from feather.